documentation/man/features/build.rst

```diff
@@ -51,6 +54,18 @@ OPTIONS
   This option allows the user to enable ``ccache`` usage during compilation
   tasks, which should improve compile times in subsequent compilations. 
 
+-w, \--warnings=(1 | 2 | 3 | 12 | 13 | 23 | 123):
+  This can be used to enable compilation warnings accordingly. Please check the
+  kernel's ``make help`` for more info.
+
+-s, \--save-log-to=path:
+  This option will save the full compilation log with the enabled warnings to
+  the specified path.
```

Comment:
Can we create a default file if the user does not provide a path? Also, we should document this behavior here.